CONTRIBUTING.md

```diff
@@ -90,8 +90,37 @@ Go ahead and push your changes to GitHub using this command.
 ```sh
 git push
 ```
-
 #### Developer Certificate of Origin (DCO) Sign off
+=======
+## Cherry-pick PRs to release branches
+
+Once you fork the project, you will notice that there are a number of branches related to Kyverno's releases such as release-1.7. You can view the list of remote branches by using the command below:
+
+```sh
+$ git branch -r
+...
+origin/release-1.5
+origin/release-1.6
+origin/release-1.7
+```
+
+Checkout one of the release branch and cherry-pick the PRs you want to merge into the release branch:
+
+```sh
+$ git checkout release-1.7
+
+git cherry-pick <commit-hash> -s
+
+git push --set-upstream origin release-1.7
+```
+
+Once the commit has been cherry picked, the author will need to open a PR merging to the release branch, release-1.7 for example.
+
+## Engage with us
```

Comment:
This section already exists on L17. Perhaps a conflict with https://github.com/kyverno/kyverno/commit/f67f145d901c9144dc6ad0c180dad0d9a2fd18e5 which moved it?